index.d.ts

```diff
@@ -0,0 +1,331 @@
+import React from 'react';
+import { ViewStyle, TextStyle } from 'react-native';
+
+export interface IOption {
+  key: any;
```

Comment:
This is passed forward to React, so the type is really `React.Attributes["key"]`, or alternatively `string | number | undefined`

Reply:
You are right, I'll put `React.Key` here